samples/peripheral/802154_phy_test/sample.yaml

```diff
@@ -9,4 +9,22 @@ tests:
       - nrf21540dk_nrf52840
       - nrf5340dk_nrf5340_cpunet
```

Comment:
Should we add nrf5340dk_nrf5340_cpuapp variant here? @ankuns @ahasztag

Reply:
No, this sample will not build for `nrf5340dk_nrf5340_cpuapp`. Only the current variant should stay